upgrade/Stage_0_Prerequisites.md

```diff
@@ -27,7 +27,7 @@ backup of Workload Manager configuration data and files is created. Once complet
 1. (`ncn-m001#`) Set the `CSM_RELEASE` variable to the **target** CSM version of this upgrade.
 
    ```bash
-   CSM_RELEASE=csm-1.2.0
+   CSM_RELEASE=csm-1.3.0
```

Comment:
In the install path, Rusty has switched over to having CSM_RELEASE be just equal to the dotted part of the release (e.g. 1.3.0). Ideally we should make that change for the upgrade path as well. It doesn't need to happen with this ticket, though. I'll open a new ticket for that.